### PR TITLE
Bump firebase/php-jwt to v6.0 for apple provider

### PR DESCRIPTION
**Pull request by boris-glumpler**
I am currently using v6 in a project, so can't install the `apple` provider atm. This change seems to have recently been made for the `eveonline` provider already, btw: https://github.com/SocialiteProviders/Providers/pull/801

### CODE REVIEW

File: src/Apple/composer.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
         "php": "^7.4 || ^8.0",
         "ext-json": "*",
         "ext-openssl": "*",
-        "firebase/php-jwt": "^5.2",
+        "firebase/php-jwt": "^6.0",
```
Comment (lucasmichot) [inline src/Apple/composer.json:37]:
```suggestion
        "firebase/php-jwt": "^5.2 || ^6.0",
```

**Review by lucasmichot — CHANGES_REQUESTED** [review]
Thanks for the PR @boris-glumpler 

I am afraid this would introduce a BC
Can you check my review ?

File: src/Apple/composer.json (hunk 1 above)
Comment (boris-glumpler) [reply src/Apple/composer.json]: Sure, that's preferable, but there wasn't much sense in it seeing that `eveonline` only requires v6.0.

**Comment by atymic** [comment]
@boris-glumpler @lucasmichot is this good to go now?

**Comment by boris-glumpler** [comment]
I made the changes, so it's up to @lucasmichot now, I guess.

**Comment by boris-glumpler** [comment]
We have decided to not implement the Apple provider after all and it seems there isn't really much enthusiasm for this PR anyways, so I am closing this now from my end. It is an easy change anyways, so can be resurrected easy enough ;)

**Pull request closed without merging**